### ISSUE TITLE
[bug]: Generator throws when found feature is only an alias

### ISSUE THREAD

**Issue by tregusti** [Financial-Times/polyfill-service-url-builder#119]
We get an error generating an URL for a rather small webpack generated file.

## What

When generating an URL for the attached file with the command in the readme (adjusted to use locally installed package instead), I get an error:

```sh
$ NODE_ENV=production BABEL_ENV=production node ./node_modules/create-polyfill-service-url/index.js analyse --file build/static/js/runtime~main.js
chrome 81.0.0 supports Promise
firefox_mob 68.0.0 supports Promise
we do not know if and_qq 1.2.0 supports Promise
chrome 81.0.0 supports document
firefox_mob 68.0.0 supports document
we do not know if and_qq 1.2.0 supports document
chrome 81.0.0 supports Object.defineProperty
firefox_mob 68.0.0 supports Object.defineProperty
we do not know if and_qq 1.2.0 supports Object.defineProperty
chrome 81.0.0 supports Symbol
firefox_mob 68.0.0 supports Symbol
we do not know if and_qq 1.2.0 supports Symbol
chrome 81.0.0 supports Symbol.toStringTag
firefox_mob 68.0.0 supports Symbol.toStringTag
we do not know if and_qq 1.2.0 supports Symbol.toStringTag
chrome 81.0.0 supports Object.create
firefox_mob 68.0.0 supports Object.create
we do not know if and_qq 1.2.0 supports Object.create
chrome 81.0.0 supports Function.prototype.bind
firefox_mob 68.0.0 supports Function.prototype.bind
we do not know if and_qq 1.2.0 supports Function.prototype.bind
chrome 81.0.0 supports console
firefox_mob 68.0.0 supports console
we do not know if and_qq 1.2.0 supports console
Failed to parse the JavaScript from xxx because an error occured:
TypeError: Cannot read property 'browsers' of undefined
    at generatePolyfillURL (C:\Users\...\node_modules\create-polyfill-service-url\src\index.js:103:52)
    at async Object.parseFile [as handler] (C:\Users\...\node_modules\create-polyfill-service-url\index.js:52:28)
```

## Details 

I have debugged the script and it seems to be failing when `describePolyfill` resolves with `undefined` on [line 102](https://github.com/Financial-Times/polyfill-service-url-builder/blob/f74bb152894599ed86183607e6d495a2d47a9090/src/index.js#L102-L103) and can't read the `browsers` property on it on line 103.

After what I can understand it is when an aliased feature is used. Two failing examples are `ArrayBuffer` and `URLSearchParams`.

We tested this in the latest version (2.0.7 and some random other 2.x versions). 1.6.0 works fine.

Our browserslist config looks like this:

```json
[
  ">0.2%",
  "not dead",
  "not ie <= 10",
  "not op_mini all"
]
```

I have tried to modify the browserslist config just to try if I can get it to work, but to no avail.

The javascript file we run against is this:

```js
!function(e){function r(r){for(var n,a,i=r[0],c=r[1],l=r[2],p=0,s=[];p<i.length;p++)a=i[p],Object.prototype.hasOwnProperty.call(o,a)&&o[a]&&s.push(o[a][0]),o[a]=0;for(n in c)Object.prototype.hasOwnProperty.call(c,n)&&(e[n]=c[n]);for(f&&f(r);s.length;)s.shift()();return u.push.apply(u,l||[]),t()}function t(){for(var e,r=0;r<u.length;r++){for(var t=u[r],n=!0,i=1;i<t.length;i++){var c=t[i];0!==o[c]&&(n=!1)}n&&(u.splice(r--,1),e=a(a.s=t[0]))}return e}var n={},o={2:0},u=[];function a(r){if(n[r])return n[r].exports;var t=n[r]={i:r,l:!1,exports:{}};return e[r].call(t.exports,t,t.exports,a),t.l=!0,t.exports}a.e=function(e){var r=[],t=o[e];if(0!==t)if(t)r.push(t[2]);else{var n=new Promise((function(r,n){t=o[e]=[r,n]}));r.push(t[2]=n);var u,i=document.createElement("script");i.charset="utf-8",i.timeout=120,a.nc&&i.setAttribute("nonce",a.nc),i.src=function(e){return a.p+"static/js/"+({0:"lodash"}[e]||e)+".chunk.js"}(e);var c=new Error;u=function(r){i.onerror=i.onload=null,clearTimeout(l);var t=o[e];if(0!==t){if(t){var n=r&&("load"===r.type?"missing":r.type),u=r&&r.target&&r.target.src;c.message="Loading chunk "+e+" failed.\n("+n+": "+u+")",c.name="ChunkLoadError",c.type=n,c.request=u,t[1](c)}o[e]=void 0}};var l=setTimeout((function(){u({type:"timeout",target:i})}),12e4);i.onerror=i.onload=u,document.head.appendChild(i)}return Promise.all(r)},a.m=e,a.c=n,a.d=function(e,r,t){a.o(e,r)||Object.defineProperty(e,r,{enumerable:!0,get:t})},a.r=function(e){"undefined"!==typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},a.t=function(e,r){if(1&r&&(e=a(e)),8&r)return e;if(4&r&&"object"===typeof e&&e&&e.__esModule)return e;var t=Object.create(null);if(a.r(t),Object.defineProperty(t,"default",{enumerable:!0,value:e}),2&r&&"string"!=typeof e)for(var n in e)a.d(t,n,function(r){return e[r]}.bind(null,n));return t},a.n=function(e){var r=e&&e.__esModule?function(){return e.default}:function(){return e};return a.d(r,"a",r),r},a.o=function(e,r){return Object.prototype.hasOwnProperty.call(e,r)},a.p="/",a.oe=function(e){throw console.error(e),e};var i=window.webpackJsonp=window.webpackJsonp||[],c=i.push.bind(i);i.push=r,i=i.slice();for(var l=0;l<i.length;l++)r(i[l]);var f=c;t()}([]);
//# sourceMappingURL=runtime~main.js.map
```

**Comment by JakeChampion** [comment]
This was fixed in #137 👍

**Comment by JakeChampion** [comment]
this was released in v2.1.0 -- https://github.com/Financial-Times/polyfill-service-url-builder/releases/tag/v2.1.0
